src/components/FilterTags/FilterTags.js

```diff
@@ -109,14 +109,14 @@ const FilterTags = () => {
 
   // Update query params when adding and removing filters
   useEffect(() => {
-    const queryParams = queryString.parse(location.search);
+    const queryParams = queryString.parse(window.location.search);
```

Comment:
This component needs to do a better job of it's queryString handling but I figured I should leave it until we switch to the new Filter UI.

Reply:
Shouldn't this use your new useQuery hook?

Reply:
Yes but this PR was already feature creeping considerably.